### PR TITLE
feat(select): add directives for user/establishment/job-qualification selection

### PR DESCRIPTION
**Pull request by GuillaumeNury**
### JobQualification

```html
<lu-simple-select jobQualifications placeholder="Placeholder..." />
<lu-multi-select jobQualifications placeholder="Placeholder..." />
```

image

image

### Establishment

```html
<lu-simple-select establishments placeholder="Placeholder..." />
<lu-multi-select establishments placeholder="Placeholder..." />
```
image

image

### User

```html
<lu-simple-select users placeholder="Placeholder..." />
<lu-multi-select users placeholder="Placeholder..." />
```

image

image

### (Bonus) MultipleSelect grouping enhancement

Now, each group have a "select all" button.

image

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2568/storybook

**Review by Supamiu — DISMISSED** [review]

File: packages/ng/core-select/input/select-input.component.ts [hunk 1]
```diff
@@ -106,6 +105,11 @@ export abstract class ALuSelectInputComponent<TOption, TValue> implements OnDest
 	public clueChanged(clue: string): void {
 		if (!this.isPanelOpen) {
 			this.openPanel(clue);
+		} else if (clue.endsWith(' ')) {
```
Comment (F2X) [inline packages/ng/core-select/input/select-input.component.ts:108]: j'ai du mal a comprendre cette condition: pourquoi on fait ce check?

Comment (GuillaumeNury) [reply packages/ng/core-select/input/select-input.component.ts]: ça tombe bien parce que ça ne sert plus !

**Review by Supamiu — DISMISSED** [review]

**Review by jeremie-lucca — APPROVED** [review]

**Review by Supamiu — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2568/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13571/ui-diff)

**Pull request merged**